code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java

```diff
@@ -151,7 +152,7 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
         return result;
     }
 
-    protected void waitForDeploymentUnit(Instance instance) {
+    protected void waitForDeploymentUnitAllocate(Instance instance) {
         // Wait until all instances in the deployment unit are out of the creating state (to ensure all of instnace.create has ran)
         if(StringUtils.isEmpty(instance.getDeploymentUnitUuid())) {
             return;
```

Comment:
I just wanted to keep all the checkpoints in the same method (handle()) for better visibility